src/search/background/index.ts

```diff
@@ -13,13 +13,15 @@ import {
     SocialSearchParams,
 } from './types'
 import { SearchError, BadTermError, InvalidSearchError } from './errors'
+import { BookmarksInterface } from 'src/bookmarks/background/types'
 
 export default class SearchBackground {
     storage: SearchStorage
     private backend
```

Comment:
Still needs type

Reply:
Very true! Though this isn't something I've added. It stands as another set of remote function implementation refactoring needed. (detailed in refactoring.md). I've added a note specifically in the code to communicate that.